src/libpostal_data

```diff
@@ -117,7 +110,7 @@ function download_file() {
         content_length=$(curl -I $url 2> /dev/null | awk '/^Content-Length:/ { print $2 }' | tr -d '[[:space:]]')
 
         if [ $content_length -ge $LARGE_FILE_SIZE ]; then
-            download_multipart $url $local_path $content_length $NUM_WORKERS
+            download_multipart $url $local_path $content_length
```

Comment:
`download_multipart` has not been using the passed argument (which becomes `num_workers`) so don't pass it.